src/screens/DialogScreen.tsx

```diff
@@ -32,7 +31,7 @@ export class DialogScreen extends React.PureComponent<
   state = {
     title: '',
     description: '',
-    onPress: emptyFunction,
+    onPress: () => null,
```

Comment:
As far I remember it was done that way so there were no issues with code coverage reports

Reply:
I will check it out later. Maybe we should use `lodash.noop` then.